src/tools/ceph_monstore_tool.cc

```diff
@@ -655,6 +655,14 @@ static int update_pgmap_meta(MonitorDBStore& st)
     t->put(prefix, "full_ratio", bl);
   }
   {
+    auto backfillfull_ratio = g_ceph_context->_conf->mon_osd_backfillfull_ratio;
+    if (backfillfull_ratio > 1.0)
+      backfillfull_ratio /= 100.0;
```

Comment:
shall we normalize the ratio in `create_initial()` and `create_pending()` if `ratio > 1.0` and spit an error message? as the document reads:

> The percentage of disk space used before an OSD is considered too ``full`` to backfill.

so in theory, any value greater than 1 is invalid.